dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -77,11 +77,7 @@ def new_pre_foorm
         end
       end
 
-      agenda = params[:agenda] || nil
-      # remove / from agenda url so module/1 => module1
-      agenda&.tr!("/", "")
-
-      render_survey_foorm(survey_name: PRE_SURVEY_CONFIG_PATHS[COURSE_BUILD_YOUR_OWN], workshop: workshop, session: nil, day: 0, workshop_agenda: agenda)
```

Comment:
As part of setting up the email to get the pre-survey link, I cleaned this up since we don't need `agenda` anymore.